standard/tier/wikis/smash/tier_data.lua

```diff
@@ -0,0 +1,139 @@
+---
+-- @Liquipedia
+-- wiki=smash
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			value = '1',
+			sort = 'A1',
+			name = 'Premier',
+			short = 'Prem.',
+			link = 'Premier Tournaments',
+			category = 'Premier Tournaments',
+		},
+		{
+			value = '2',
+			sort = 'A2',
+			name = 'Major',
+			short = 'Maj.',
+			link = 'Major Tournaments',
+			category = 'Major Tournaments',
+		},
+		{
+			value = '3',
+			sort = 'A3',
+			name = 'National',
+			short = 'Nat.',
+			category = 'National Tournaments',
+		},
+
+		-- legacy??? to be replaced by `-1` ???
+		[4] = {
+			value = '4',
+			sort = 'A4',
+			name = 'No Tier',
+			short = 'None',
+			link = 'No Tier Tournaments',
+			category = 'No Tier Tournaments',
+		},
+		-- legacy??? to be replaced by `-1` ???
+		[-99] = {
+			value = '-99',
+			sort = 'D1',
+			name = 'Undefined',
+			short = '?',
+		},
+		-- legacy??? to be replaced by `-1` ???
+		[''] = {
+			value = '-99',
+			sort = 'D1',
+			name = 'Undefined',
+			short = '?',
+		},
+	},
+
+	tierTypes = {
+		invitational = {
+			value = 'Invitational',
+			sort = 'B1',
+			name = 'Invitational',
+			short = 'Inv.',
+			category = 'Invitational Tournaments',
+		},
+		monthly = {
+			value = 'Monthly',
+			sort = 'B2',
+			name = 'Monthly',
+			short = 'Mon.',
+			category = 'Monthly Tournaments',
+		},
+		['bi-monthly'] = {
```

Comment:
Grammatical consistency. Bi-monthly has a dash whereas biweekly doesn't

Reply:
i retrieved the currently set values via an lpdb query
if we adjust it here the pages should be adjusted too

Reply:
That's fine, this is easily botted and there should be very few of this tier type

Reply:
alright, letting the botting for you then^^